lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1814,7 +1814,7 @@ namespace Js
             // Free unused bytes
             Assert(cbNeeded + 1 <= cbUtf8Buffer);
             *ppSourceInfo = Utf8SourceInfo::New(this, utf8Script, (int)length,
-                cbNeeded, pSrcInfo, isLibraryCode, scriptSource);
```

Comment:
Can we remove the scriptSource argument from the definition too?  It serves no purpose if we always copies